modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -145,7 +167,7 @@ export default {
             all: 1
           }
         }
-      )).results;
```

Comment:
I'm not sure when this changed.

Reply:
Probably a buggy borrowing from pieces, which return metadata related to pagination etc. as well as the pieces.